src/main/java/com/gmail/val59000mc/scenarios/Scenario.java

```diff
@@ -54,6 +54,7 @@
     NOGOINGBACK("No Going Back", UniversalMaterial.NETHER_BRICK, null, "&6No Going Back&7:", "&7- Once you go through a nether portal in the overworld, you can't go back to the overworld."),
     DOUBLEDATES("Double Dates", UniversalMaterial.RED_BANNER, DoubleDatesListener.class, "&6Double Dates&7:", "&7- Players join with a chosen team and are combined randomly with another team."),
     FLYHIGH("Fly High", UniversalMaterial.ELYTRA, FlyHighListener.class, 9, "&6Fly High&7:", "&7- Players spawn with a elytra."),
+    RANDOMITEMS("Random Items", UniversalMaterial.EXPERIENCE_BOTTLE, RandomizedBlockDropsListener.class, "&Random Drops!&7:", "&7-Every block will drop a random item when broken." ),
```

Comment:
Change the scenario enum from "RANDOMIZEDDROPS" to "RANDOMITEMS".
Fix the color formatting of the scenario name. ("&Random Drops!&7:" -> "&6Random Drops&7:").
Add a space between the - and the first word of the description.